src/bootstrap/format.rs

```diff
@@ -34,12 +34,21 @@ pub fn format(build: &Build, check: bool) {
     }
 
     {
-        let mut cmd = Command::new(&cargo_fmt_path);
+        let mut cmd = Command::new(&rustfmt_path);
         cmd.env("PATH", std::env::join_paths(&paths).unwrap())
             .current_dir(&build.src)
-            .arg("fmt")
-            .arg("--version");
-        dbg!(cmd.status().unwrap());
+            .arg("--edition").arg("2018").arg("--verbose").arg("--check")
```

Comment:
Should the edition flag move to rustfmt.toml?

Reply:
All of these latest commits are very much just testing things -- I wouldn't expect them to stick around.